components/admin/hpc-workspace/SPECS/hpcworkspace.spec

```diff
@@ -0,0 +1,122 @@
+#----------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+%include %{_sourcedir}/OHPC_macros
+
+%define pname hpc-workspace
+
+Name:    %{pname}%{PROJ_DELIM}
+Version: 1.4.0
+Release: 1%{?dist}
+Summary: Temporary workspace management
+License: GPL
+Source:  https://github.com/holgerBerger/hpc-workspace/archive/refs/tags/%{version}.tar.gz
+Source1: ws.conf.template
+Group:   %{PROJ_NAME}/admin
+URL:     https://github.com/holgerBerger/hpc-workspace
+
+BuildRequires: gcc-c++
+BuildRequires: python3-devel
+BuildRequires: boost-devel
+BuildRequires: yaml-cpp-devel
+BuildRequires: ncurses-devel
+BuildRequires: cmake
+Requires: python3
+Requires: python3-pyyaml
+Requires: lua5.1
+Requires: boost-system
+Requires: boost-filesystem
+Requires: boost-regex
+Requires: boost-program-options
+Requires: ncurses
+Requires: libcap
+Requires: yaml-cpp
+
+
+# Default library install path
+%define install_path %{OHPC_ADMIN}/%{pname}/%{version}
+
+%description
+HPC Workspace provides tools to provision temporaty workspaces on scratch or 
+working filesystems in an HPC environment.
+
+%prep
+%setup -q -n %{pname}-%{version}
+
+%build
+cmake \
+  -DCMAKE_INSTALL_PREFIX=%{install_path} \
+  .
+make -j 4
```

Comment:
Please do not hard code the number of parallel tasks to 4. There is a macro for this.

Reply:
Resolved in ca83914